### ISSUE TITLE
[Feature Request] Clean up unused volumes

### ISSUE THREAD

**Issue by dennisschott** [caprover/caprover#428]
Could you add a clean up for unused volumes? When serveral apps with persistent storage manages the volumes by it's own, it can be very messy in "/var/lib/docker/volumes/".

**Comment by githubsaturn** [comment]
What version of CapRover are you running?

Starting v1.4 you can delete volumes when you're deleting apps. So the cleanup is being done during app removal.

**Comment by dennisschott** [comment]
Yes, you are right, but an option for an still existing app would be awesome.

**Comment by githubsaturn** [comment]
I'm trying to understand the use case for this? You have an existing app and you want to remove its volumes? This will make the app crash.

**Comment by dennisschott** [comment]
Off course, but when you switch the name of a persistent storage the old storage will remain.

**Comment by githubsaturn** [comment]
Renaming a volume seems like an odd and not-very-common procedure to me... But I'll keep this issue to open to hear feedback from others as well.

**Comment by BamButz** [comment]
I would support this, but not because I rename my volumes, but if I want to reset the application, I have to delete the app and re-declare all environment variables and possible NGINX configurations.

With the ability to remove volumes, I can simply delete the persisted data and recreate the volume.

**Comment by SteffenL** [comment]
I found myself in a situation where I had an abandoned volume that wouldn't be deleted automatically, but I came up with a couple simple workarounds so it wasn't a big deal.

## Steps to reproduce:

1. Create app `my-app` with persistent data.
2. Add persistent directory: `/home/user/data`, `my-app-data`
3. Rename persistent directory: `/home/user/data`, `my-app-data2`
4. Volumes `captain--my-app-data` and `captain--my-app-data2` are now present under `/var/lib/docker/volumes`.
5. `my-app-data` can no longer be deleted automatically along with `my-app`.

## Workaround using the CapRover web UI:

1. Create app `my-temp-app` with persistent data.
2. Add persistent directory: `/home/user/data`, `my-app-data`
3. Delete `my-temp-app` and let `my-app-data` be deleted along with it.
4. Volume `captain--my-app-data` is no longer present under `/var/lib/docker/volumes`.

## Workaround on the CapRover host:

1. `docker volume rm captain--my-app-data`
2. Volume `captain--my-app-data` is no longer present under `/var/lib/docker/volumes`.
